### DISCUSSION TITLE
Groups and AllowTo configuration of lotus-worker

### DISCUSSION THREAD

**Discussion by ifanheng** [filecoin-project/lotus#8147] [General Q&A]
lotus-worker version: 1.15.1-dev+calibnet+git.bac708fa3

Did I miss any configuration?
I want PC1 and PC2 to be bound to one machine, but PC2 always fails.
 
This is my configuration of lotus-worker:
$ cat /lotus/worker/p/sectorstore.json
{
  "ID": "e0f63f9f-a80f-4057-84d6-8971a7e43f0a",
  "Weight": 10,
  "CanSeal": true,
  "CanStore": false,
  "MaxStorage": 0,
  "Groups": ["c82-1"],
  "AllowTo": ["c82-1"]
}

the PC2 failed log：
13.     2022-02-18 11:40:55 +0800 CST:  [event;sealing.SectorSealPreCommit2Failed]      {"User":{}}
        seal pre commit(2) failed: storage call error 0: checking PreCommit failed: Rust panic: no unwind information
14.     2022-02-18 11:41:55 +0800 CST:  [event;sealing.SectorRetrySealPreCommit2]       {"User":{}}
15.     2022-02-18 14:04:03 +0800 CST:  [event;sealing.SectorSealPreCommit2Failed]      {"User":{}}
        seal pre commit(2) failed: storage call error 0: checking PreCommit failed: Rust panic: no unwind information
16.     2022-02-18 14:05:03 +0800 CST:  [event;sealing.SectorRetrySealPreCommit2]       {"User":{}}
17.     2022-02-18 15:08:57 +0800 CST:  [event;sealing.SectorSealPreCommit2Failed]      {"User":{}}
        seal pre commit(2) failed: storage call error 0: checking PreCommit failed: Rust panic: no unwind information
18.     2022-02-18 15:09:57 +0800 CST:  [event;sealing.SectorRetrySealPreCommit2]       {"User":{}}

**Comment by SgtCoin** [comment]
Hi @ifanheng,

I believe this is the same as https://filecoinproject.slack.com/archives/CEGN061C5/p1645169943070979

If that is the case, have you changed to a stable release from master? If so, did it correct the issue?
